scripts/crosstool-NG.sh

```diff
@@ -616,6 +616,11 @@ if [ -z "${CT_RESTART}" ]; then
     CT_DoExecLog ALL mkdir -p "${CT_PREFIX_DIR}/bin"
     CT_DoExecLog DEBUG ${install} -m 0755 "${CT_LIB_DIR}/scripts/toolchain-config.in" "${CT_PREFIX_DIR}/bin/${CT_TARGET}-ct-ng.config"
     CT_DoExecLog DEBUG ${sed} -i -e 's,@@grep@@,"'"${grep}"'",;' "${CT_PREFIX_DIR}/bin/${CT_TARGET}-ct-ng.config"
+    if [ -n "$CT_LIBC_UCLIBC_CONFIG_FILE" ]
+    then
+        ${install} -m 0755 "${CT_PREFIX_DIR}/bin/${CT_TARGET}-ct-ng.config" "${CT_PREFIX_DIR}/bin/${CT_TARGET}-libc.config"
```

Comment:
I'd prefer not to burden the main driver (crosstool-NG.sh) with actions pertinent only to a particular type of C library. Please do this in uClibc.sh (e.g. in `uClibc_post_cc`).